docs/api/attachments.md

```diff
@@ -0,0 +1,176 @@
+# Attachments
+Supports adding, viewing, deleting, and updating attachments.
+
+## Description
+Attachments are used to support any kind of external file. These are often PDFs or images. Attachments are used for Material Transfer Agreements, Institutional Review Board documents, and sequence maps for materials among other things.
+
+## Add a new attachment:
+
+**Request**:
+
+`POST` `/attachments/`
+
+Parameters:
+
+Name            | Type   | Description
+----------------|--------|---
+filename        | string | The filename of the attachment.
+description     | string | A description of the attachment.
+s3_bucket       | string | The url of the s3_bucket where your attachment file is stored.
```

Comment:
This should also be changed to say bucket name.

Reply:
Done.